bdb/info.c

```diff
@@ -1665,9 +1671,21 @@ void bdb_process_user_command(bdb_state_type *bdb_state, char *line, int lline,
         }
         host = tokdup_hostname(host, ltok);
         process_add(bdb_state, host);
-    }
-
-    else if (tokcmp(tok, ltok, "del") == 0) {
+    } else if (tokcmp(tok, ltok, "hostname") == 0) {
+        char *iHost, *oHost;
+        Pthread_mutex_lock(&hostname_lookup_lk);
+        gbl_verbose_hostname_lookup = 1;
+        iHost = segtok(line, lline, &st, &ltok);
+        if (ltok == 0) {
+            logmsg(LOGMSG_USER, "expected hostname\n");
+            Pthread_mutex_unlock(&hostname_lookup_lk);
+            return;
```

Comment:
I think maybe set `gbl_verbose_hostname_lookup` to 0 here.

Reply:
Done!

Reply:
It does not seem that the  `gbl_verbose_hostname_lookup `  flag protects anything, though. Do we need the mutex here, too?